README.md

```diff
@@ -0,0 +1,121 @@
+![IPFS Logo] (ipfs.png)
+
+
+[![Join the chat at https://gitter.im/ipfs/py-ipfs](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ipfs/py-ipfs?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
```

Comment:
Add banners? See the [Readme in ipfs/ci-status](https://github.com/ipfs/ci-status), which checks for inclusion of these links.